content/en/docs/glossary/_index.md

```diff
@@ -32,7 +32,11 @@ In the context of Kubernetes, a Custom Resource (CR) is an extension mechanism t
 
 A Custom Resource Definition (CRD) is an extension mechanism that allows users to define their own custom resources. It enables the creation of new resource types with specific schemas and behaviors. CRDs define the structure and validation rules for custom resources, allowing users to store and manage additional information beyond the built-in Kubernetes resources. Once a CRD is created, instances of the custom resource can be created, updated, and deleted using the Kubernetes API. CRDs provide a way to extend Kubernetes and tailor it to specific application requirements.
 
-For example, the following `.yaml` file describes a `SpinApp` using CRD syntax:
+## SpinAppCRD
+
+The SpinApp CRD is a Kubernetes resource that extends the functionality of the Kubernetes API to support Spin applications. It defines a custom resource called "SpinApp" that encapsulates all the necessary information to deploy and manage a Spin application within a Kubernetes cluster. The SpinAppCRD consists of several key fields that define the desired state of a Spin application. 
+
+Here's an example of a SpinAppCRD definition:
```

Comment:
Pedantic update - difference between CRD and custom resources (or objects) that are based on that CRD.  Perhaps something like the following?

```suggestion
Here's an example of a SpinApp custom resource that uses the SpinApp CRD schema:
```